pajbot/modules/casechecker.py

```diff
@@ -37,33 +36,99 @@ class CaseCheckerModule(BaseModule):
             constraints={"min_value": 100, "max_value": 1000},
         ),
         ModuleSetting(
-            key="timeout_duration",
-            label="Timeout duration",
+            key="lowercase_timeouts", label="Enable lowercase timeouts", type="boolean", required=True, default=False
+        ),
+        ModuleSetting(
+            key="lowercase_timeout_duration",
+            label="Lowercase Timeout duration",
             type="number",
             required=True,
             placeholder="",
             default=3,
             constraints={"min_value": 3, "max_value": 120},
         ),
         ModuleSetting(
-            key="online_chat_only", label="Only enabled in online chat", type="boolean", required=True, default=True
+            key="lowercase_timeout_reason",
+            label="Lowercase Timeout Reason",
+            type="text",
+            required=False,
+            placeholder="",
+            default="Too many lowercase characters ",
```

Comment:
you accidentally put a trailing space here
```suggestion
            default="Too many lowercase characters",
```